src/tests/fillPdf.test.ts

```diff
@@ -0,0 +1,34 @@
+import { fillPDF } from 'ustaxes/pdfFiller/fillPdf'
+import { downloadPDF } from '../pdfFiller/pdfHandler'
+
+describe('fillPdf', () => {
+  it('should fill PDF according value array', async () => {
+    const pdf = await downloadPDF('https://ustaxes.org/forms/f1040.pdf')
```

Comment:
I ran into a similar issue adding tests for PDFs, so I was able to do some refactoring to avoid downloading from the .org in #717. I'll revisit this PR after that one is merged.